MediaBrowser.Model/Dlna/StreamBuilder.cs

```diff
@@ -1009,18 +1019,17 @@ private void BuildStreamVideoItem(
 
             var appliedVideoConditions = options.Profile.CodecProfiles
                 .Where(i => i.Type == CodecType.Video &&
-                    i.ContainsAnyCodec(videoCodecs, container, useSubContainer) &&
+                    i.ContainsAnyCodec([.. videoCodecs], container, useSubContainer) &&
                     i.ApplyConditions.All(applyCondition => ConditionProcessor.IsVideoConditionSatisfied(applyCondition, width, height, bitDepth, videoBitrate, videoProfile, videoRangeType, videoLevel, videoFramerate, packetLength, timestamp, isAnamorphic, isInterlaced, refFrames, numVideoStreams, numAudioStreams, videoCodecTag, isAvc)))
                 // Reverse codec profiles for backward compatibility - first codec profile has higher priority
                 .Reverse();
-
-            foreach (var i in appliedVideoConditions)
+            foreach (var condition in appliedVideoConditions)
             {
                 foreach (var transcodingVideoCodec in videoCodecs)
```

Comment:
Ideally, use the same array. I probably should have suggested it to Nyan right away.
```suggestion
                foreach (var transcodingVideoCodec in playlistItem.VideoCodecs)
```